src/components/core/icons.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const IconDrag = () => (
```

Comment:
Isn't there an equivalent in `ui-icons`?

Reply:
No, @cooper-joe will add one later.